gui/gui.go

```diff
@@ -587,3 +591,17 @@ func (gui *GUI) SwapBuffers() {
 	UpdateNSGLContext(gui.window)
 	gui.window.SwapBuffers()
 }
+
+func (gui *GUI) Screenshot(path string) {
```

Comment:
Same thing - should we move the `Screenshot()` function to `gui_test.go` file?

Reply:
It might be cool to be able to bind keys to this to take screenshots of the terminal and dump them to the users desktop as a future feature?

Reply:
@liamg I believe so as well! (even have though about that too! :) )

Reply:
@nikitar020 `Screenshot()` is not accessible from `main_test.go` as well if moved to `gui_test.go`